sdk/src/azure/core/az_mqtt5_rpc_server_hfsm.c

```diff
@@ -69,8 +69,8 @@ static az_result root(az_event_policy* me, az_event event)
       }
       else
```

Comment:
I think this whole else can be removed now that there is an inbound policy (it should do the same thing as the if statement above it). I can investigate this

Reply:
(same for other root state AZ_HFSM_EVENT_ERRORs)

Reply:
Changed! Good point, also looked into the other files